docs(flutter): Update debug symbols page (#16273)
getsentry/sentry-docs: docs/platforms/dart/guides/flutter/debug-symbols/dart-plugin.mdx

sidebar_order: 1
---

> The [Sentry Dart Plugin](https://github.com/getsentry/sentry-dart-plugin) is the recommended way to upload debug symbols for Flutter applications. It uploads debug symbols and source maps to Sentry which supports:

Reviewer: ```suggestion
The [Sentry Dart Plugin](https://github.com/getsentry/sentry-dart-plugin) is the recommended way to upload debug symbols for Flutter applications. It uploads debug symbols and source maps to Sentry, this enables:
```
